templates/index.html

```diff
@@ -50,8 +49,90 @@ <h1>Very fast server provisioning for your data centre</h1>
 
-<section class="p-strip">
+<section class="p-strip u-fixed-width">
+  <div class="p-strip is-shallow u-no-padding--top">
+    <h2>Data centre automation</h2>
+    <h4>Your next rack is a bare metal cloud.<br>Metal-As-A-Service (MAAS) provisioning with Windows, ESXi, Linux.</h4>
+  </div>
+  <ul class="p-list is-split">
+    <li class="p-list__item is-ticked">Bare metal cloud with on-demand servers</li>
+    <li class="p-list__item is-ticked">Remote edge cluster operations</li>
+    <li class="p-list__item is-ticked">Infrastructure monitoring and discovery</li>
+    <li class="p-list__item is-ticked">Ansible, Chef, Puppet, SALT, Juju integration</li>
+    <li class="p-list__item is-ticked">Super fast install from scratch</li>
+    <li class="p-list__item is-ticked">VMWare ESXi, Windows, CentOS, RHEL, Ubuntu</li>
+    <li class="p-list__item is-ticked">Custom images with pre-installed apps</li>
+    <li class="p-list__item is-ticked">Disk and network configuration</li>
+    <li class="p-list__item is-ticked">API-driven DHCP, DNS, PXE, IPAM</li>
+    <li class="p-list__item is-ticked">REST API for provisioning</li>
+    <li class="p-list__item is-ticked">LDAP user authentication</li>
+    <li class="p-list__item is-ticked">Role-based access control (RBAC)</li>
+    <li class="p-list__item is-ticked">Hardware testing and commissioning</li>
+  </ul>
+  <ul class="row u-no-margin--bottom p-strip is-shallow">
+    <li class="col-2 col-medium-1 col-small-1">
+      <img src="https://assets.ubuntu.com/v1/59dc9048-verizon.png" alt="Verizon logo">
+    </li>
+    <li class="col-2 col-medium-1 col-small-1">
+      <img src="https://assets.ubuntu.com/v1/88730cec-Telefonica.png" alt="Telefonica logo">
+    </li>
+    <li class="col-2 col-medium-1 col-small-1">
+      <img src="https://assets.ubuntu.com/v1/b8f78004-Tele2.png" alt="Tele2 logo">
+    </li>
+    <li class="col-2 col-medium-1 col-small-1">
+      <img src="https://assets.ubuntu.com/v1/e0fec57a-Telecom+Italia.png" alt="Telecom Italia logo">
+    </li>
+    <li class="col-2 col-medium-1 col-small-1 col-start-small-2">
+      <img src="https://assets.ubuntu.com/v1/08a27e4a-NEC.png" alt="NEC logo">
+    </li>
+    <li class="col-2 col-medium-1 col-small-1 col-start-small-3">
+      <img src="https://assets.ubuntu.com/v1/d44c4c5e-Barclays.png" alt="Barclays logo">
+    </li>
+  </ul>
+  <hr>
+  <div class="u-align--right">
+    <ul class="p-inline-list">
+      <li class="p-inline-list__item">
+        <a class="p-button--base js-invoke-modal" href="/contact-us" onclick="dataLayer.push({
+              'event': 'GAEvent',
+              'eventAction': 'Click' ,
+              'eventCategory': 'Data centre automation',
+              'eventLabel' : 'Contact us'
+            });"> Contact us </a>
+      </li>
+      <li class="p-inline-list__item last-item">
+        <a class="p-button--positive u-no-margin--right" href="/install">Install today</a>
+      </li>
+    </ul>
+  </div>
+  </div>
```

Comment:
Too many closing divs here which is closing the `#main-content` div here.

Reply:
fixed